DataFormats/TrackerCommon/interface/TrackerTopology.h

```diff
@@ -257,13 +258,30 @@ class TrackerTopology {
   bool tecIsZPlusSide(const DetId &id) const {return !tecIsZMinusSide(id);}
   bool tecIsZMinusSide(const DetId &id) const { return tecSide(id)==1;}
 
-  //these are from the old TOB/TEC/TID/TIB DetId
+  //these are from the old TOB/TEC/TID/TIB DetId: should NOT be used anymore!!
   bool tobIsStereo(const DetId &id) const {return tobStereo(id)!=0 && !tobIsDoubleSide(id);}
   bool tecIsStereo(const DetId &id) const {return tecStereo(id)!=0 && !tecIsDoubleSide(id);}
   bool tibIsStereo(const DetId &id) const {return tibStereo(id)!=0 && !tibIsDoubleSide(id);}
   bool tidIsStereo(const DetId &id) const {return tidStereo(id)!=0 && !tidIsDoubleSide(id);}
 
-  //these are clones of the old SiStripDetId
+  bool tobIsRPhi(const DetId &id) const { return SiStripDetId(id).stereo()==0 && !tobIsDoubleSide(id);}
```

Comment:
TrackerTopology can not depend on any of the tracker DetId classes. Use tobIsStereo instead (for example)